plugin/commands.go

```diff
@@ -0,0 +1,123 @@
+package plugin
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/pkg/errors"
+	"gopkg.in/urfave/cli.v1"
+)
+
+// CommandPlugin is definition of mkr plugin
+var CommandPlugin = cli.Command{
+	Name:        "plugin",
+	Usage:       "Manage mackerel plugin",
+	Description: `[WIP] Manage mackerel plugin`,
+	Subcommands: []cli.Command{
+		{
+			Name:        "install",
+			Usage:       "install mackerel plugin",
+			Description: `WIP`,
+			Action:      doPluginInstall,
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "prefix",
+					Usage: "plugin install location",
+				},
+			},
+		},
+	},
+	Hidden: true,
+}
+
+// main function for mkr plugin install
+func doPluginInstall(c *cli.Context) error {
+	argInstallTarget := c.Args().First()
```

Comment:
- It's nice if `mkr plugin install` supports installing multiple plugins at once as `mkr plugin install somePlugin2 somePlugin2`, like `apt-get` and  `yum`  (and more) does
- Is there any objection to use `len(c.Args())` to check presence of install target?